ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1644,3 +1644,45 @@ def _is_frontend_array(x):
         or isinstance(x, tf_tensor)
         or isinstance(x, DeviceArray)
     )
+
```

Comment:
I believe this is a helper function that is unique to the `test_tensorflow_deseralize()` function. In these cases, It's best to implement such functions in the test file itself rather than in the helpers. Refer to the following section of the guide for more details.
https://lets-unify.ai/ivy/deep_dive/ivy_frontends_tests.html#hypothesis-helpers